CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 ------------------------------------------------------------------------
 - Fix: [#2379] Text cursor is not rendering in the right position.
 - Fix: [#2380] 'Number of smoothing passes' label is displayed incorrectly.
+- Fix: [#2387] Crash when viewing vehicle orders.
```

Comment:
This doesn't always happen, though. Can we make it more specific?

Reply:
I could copy the text from 2127

Reply:
For reference, #2127 added:
> Crash when viewing the orders of a vehicle and ai deletes a train.

That sounds more specific alright. Can I suggest we generalise it to:
```suggestion
- Fix: [#2387] Potential crash when a vehicle is deleted while viewing vehicle orders.
```